src/components/section/Section.tsx

```diff
@@ -34,6 +34,27 @@ const Section: FunctionComponent<Props> = ({ page, section }) => {
         </SectionStack>
       )
 
+    case 'SectionTitle':
```

Comment:
Reminder: @jtfairbank & @Panzemek to make github issues to guide the creation of each of these section types as Contentful Models & React Components.